### ISSUE TITLE
Tags should not be listed as inactive branches

### ISSUE THREAD

**Issue by stof** [travis-ci/travis-ci#4889]
Currently, the ``branches`` page includes tags as if they were branches (and so inactive branches in the new UI). Tag builds should be treated differently IMO

**Comment by joshk** [comment]
You are correct, this should not happen, but will require some extra changes from us behind the scenes.

I will keep this open for now as we should look into fixing this.

**Comment by ELLIOTTCABLE** [comment]
:+1:!

(Also, it's unclear how branches become ‘inactive’ branches. I have branches with no commits for many months that are still clogging up ‘active branches.’ o_O)

**Comment by joshk** [comment]
We will look at changing 'Inactive Branches' to 'Removed Branches' (eg. removed from GitHub)

**Comment by RuCray** [comment]
I have a long running branch that I am still making changes to that's marked as inactive.  There's no indication that Travis is running when I push to that branch.  It would be nice to have it fixed.

**Comment by ELLIOTTCABLE** [comment]
Yeah, between @RuCray's situation, and my own approaches (I have no intention of removing branches from GitHub: old branches have unique commits, with important history and comments, that never get merged to future-proof moving-forward branches due to squashing), I'd really prefer to see ‘active/inactive’ be a manually-toggleable setting.

**Comment by peccolor** [comment]
this problem is still present after 2 years: can you add a toggable setting for it?
